src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
   </tbody>
 </table>
 
-### Correlation statistics [#correlation]
+### Use suggested decisions [#suggested-decisions]
 
-Whether you want to use the suggested decisions we provide using our pattern recognition algorithms, or you want to add in your own correlation logic, you'll get insights into your correlation rate, noise reduction improvement, and the number of correlated issues arising.
+The data from your selected sources is continuously inspected for patterns to help reduce noise. Once patterns have been observed in your data, decisions are suggested that would allow these events to correlate in the future.
 
-From the UI, you can view the [underlying NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) and create your own custom charts and dashboards from this data.
+To get started, click **Suggested decisions** panel on the topic of **Decisions** UI page. You can see the logic behind the suggested decision, and the estimated correlation rate by clicking **Why was this suggested**.
 
 <img
-  title="Decisions Statistic Block"
-  alt="A screenshot of a decisions statistic block"
-  src={iiDecisionsStatistics}
+  title="Suggested decision block"
+  alt="A screenshot of a suggested decision block"
+  src={screenShot20200727At65041Am}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Some example statistics from the decisions UI.
 </figcaption>
 
-Some definitions of statistics:
+To enable the suggested decision, click **Activate decision**. The decision will appear in your decision collection marked Creator as New Relic AI with a robot icon. If the decision isn't relevant to your needs, click **Dismiss**.
 
-* Correlation rate: The percentage of the time correlations are occurring versus not.
-* Total correlated issues: Number of issues correlated with another issue.
-* Noise reduction: Total number of issues after correlation divided by the total number of issues before correlation.
-* Correlation reason: Shows which decisions are correlating issues the most.
+## Create custom decisions [#customize]
 
-### Use suggested decisions [#suggested-decisions]
+You can reduce noise and improve correlation by building your own custom decisions. To start building a decision, go to [one.newrelic.com](https://one.newrelic.com) &gt; **Alerts & AI** &gt; **Correlate** &gt; **Decisions**, and click **Add a decision**.
+
+*Logic filter: Logic condition defined with an [operator](#operators) on an [attribute](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/violation-event-attributes/#attributes).
```

Comment:
usually for a bulletpoint list, we want some sort of intro to them. it's not clear to me what this bulletpoint list is or why it's important.